rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec

```diff
@@ -0,0 +1,68 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_foreman_openscap
+%global confdir hammer
+
+Summary: Foreman Openscap commands for Hammer CLI
```

Comment:
Should be "OpenSCAP" - note that this doesn't match the summary in the gemspec, they ought to be the same.